src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/13.diff.sql

```diff
@@ -0,0 +1,424 @@
+    BEGIN TRANSACTION
+
+    declare commands cursor for
+    SELECT N'ALTER INDEX [' + indexes.IndexName + '] ON ' + indexes.TableName + ' Rebuild;'
+    FROM @indexes as indexes
+
+    declare @cmd varchar(max)
+
+    open commands
+    fetch next from commands into @cmd
+    while @@FETCH_STATUS=0
+    begin
+      exec(@cmd)
+      fetch next from commands into @cmd
+    end
+
+    select indexName, tableName from @indexes
+
+    COMMIT TRANSACTION
+GO
+
+
+/*************************************************************
+    Stored procedures for remove duplicate resources
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     DeleteDuplicatedResources
+--
+-- DESCRIPTION
+--     Delete duplicated resources
+--
+-- PARAMETERS
+--
+CREATE OR ALTER PROCEDURE dbo.DeleteDuplicatedResources
+AS
+    SET NOCOUNT ON
+    SET XACT_ABORT ON
+
+    SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+    BEGIN TRANSACTION
+
+    DELETE rank FROM
+		(
+			SELECT *
+			, DupRank = ROW_NUMBER() OVER (
+						  PARTITION BY ResourceId
+						  ORDER BY ResourceSurrogateId desc)
+			From dbo.Resource
+		) as rank
+    where rank.DupRank > 1
+
+    COMMIT TRANSACTION
+GO
+
+/*************************************************************
+    Stored procedures for remove duplicate search parameters
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     DeleteDuplicatedSearchParams
+--
+-- DESCRIPTION
+--     Delete duplicated search parameters
+--
+-- PARAMETERS
+--     @tableName
+--         * search params table name
+CREATE OR ALTER PROCEDURE dbo.DeleteDuplicatedSearchParams
+    @tableName nvarchar(128)
+AS
+    SET NOCOUNT ON
+    SET XACT_ABORT ON
+
+    SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+    BEGIN TRANSACTION
+
+	DECLARE @Sql NVARCHAR(MAX);
+
+	SET @Sql = N'DELETE FROM ' + @TableName
+	+ N' WHERE ResourceSurrogateId not IN (SELECT ResourceSurrogateId FROM dbo.Resource)'
+
+	EXECUTE sp_executesql @Sql
+
+    COMMIT TRANSACTION
+GO
+
+/*************************************************************
+    Stored procedures for general task
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     CreateTask_2
+--
+-- DESCRIPTION
+--     Create task for given task payload.
+--
+-- PARAMETERS
+--     @taskId
+--         * The ID of the task record to create
+--     @queueId
+--         * The number of seconds that must pass before an export job is considered stale
+--     @taskTypeId
+--         * The maximum number of running jobs we can have at once
+--     @maxRetryCount
+--         * The maximum number for retry operation
+--     @inputData
+--         * Input data payload for the task
+--     @isUniqueTaskByType
+--         * Only create task if there's no other active task with same task type id
+--
+CREATE OR ALTER PROCEDURE [dbo].[CreateTask_2]
+    @taskId varchar(64),
+    @queueId varchar(64),
+	@taskTypeId smallint,
```

Comment:
Nit: this file has a mix of spaces and tabs -- could we please convert all tabs to spaces?

Reply:
Sure, update. Thanks.